office-365-feed/playbooks/playbooks.json

```diff
@@ -0,0 +1,565 @@
+      "workflows": [
+        {
+          "@type": "Workflow",
+          "triggerLimit": null,
+          "name": "> Office 365 Feed > Fetch",
+          "aliasName": null,
+          "tag": "#office-365-feed #dataingestion #fetch",
+          "description": "Data Ingestion Playbook",
+          "isActive": false,
+          "debug": false,
+          "singleRecordExecution": false,
+          "remoteExecutableFlag": false,
+          "parameters": [
+            "last_pull_time"
+          ],
+          "synchronous": false,
+          "lastModifyDate": 1642086249,
+          "collection": "/api/3/workflow_collections/c4468567-68d9-416d-aaf5-727209d904b1",
+          "versions": [],
+          "triggerStep": "/api/3/workflow_steps/a8d633e4-5ee7-4e45-b8fb-d85d2430410a",
+          "steps": [
+            {
+              "@type": "WorkflowStep",
+              "name": "Start",
+              "description": null,
+              "arguments": {
+                "step_variables": {
+                  "input": {
+                    "params": []
+                  },
+                  "_configuration_schema": "[\n  {\n    \"title\": \"Type of feeds to ingest\",\n    \"name\": \"feedtypes\",\n    \"type\": \"select\",\n    \"tooltip\": \"Select one or more types you want to pull from the MalSilo feeds list\",\n    \"required\": true,\n    \"editable\": true,\n    \"visible\": true,\n    \"options\": [\n      \"IPs\",\n      \"URLs\",\n      \"Both\"\n    ]\n  },\n  {\n    \"title\": \"Confidence\",\n    \"name\": \"confidence\",\n    \"type\": \"integer\",\n    \"tooltip\": \"Default Confidence score to be set for the feeds\",\n    \"required\": false,\n    \"editable\": true,\n    \"visible\": true,\n    \"value\": 50,\n    \"validation\": {\n      \"minlength\": 0,\n      \"maxlength\": 100\n    }\n  },\n  {\n    \"title\": \"Reputation\",\n    \"name\": \"reputation\",\n    \"type\": \"select\",\n    \"tooltip\": \"Default Reputation to be set for the feeds\",\n    \"required\": true,\n    \"editable\": true,\n    \"visible\": true,\n    \"options\": [\n      \"Good\",\n      \"Suspicious\",\n      \"Malicious\",\n      \"No Reputation Available\",\n      \"TBD\"\n    ],\n    \"value\": \"Suspicious\"\n  },\n  {\n    \"title\": \"TLP\",\n    \"name\": \"tlp\",\n    \"type\": \"select\",\n    \"tooltip\": \"Default TLP to be set for the feeds\",\n    \"required\": false,\n    \"editable\": true,\n    \"visible\": true,\n    \"options\": [\n      \"Red\",\n      \"Amber\",\n      \"Green\",\n      \"White\"\n    ],\n    \"value\": \"White\"\n  },\n  {\n    \"title\": \"Maximum Age (in days)\",\n    \"name\": \"expiry\",\n    \"type\": \"integer\",\n    \"tooltip\": \"Default age for the feeds. The indicator expires on 'valid_until' date unless a different expiry is specified here. If specified, the indicator will be marked expired on valid_from + expiry.\",\n    \"required\": false,\n    \"editable\": true,\n    \"visible\": true\n  }\n]"
```

Comment:
Default Reputation: Good, Since this is a whitelist
TLP: White, since this is open api
Confidence: 90 - static list of MS IPs and URLs
Expiry: 30 - since it is refreshed monthly